packages/Training/src/Repository/TrainingRepository.php

```diff
@@ -40,6 +40,7 @@ public function fetchRecentlyActive(): array
             ->setParameter(':weekAgo', DateTime::from('- 7 days'))
             ->setParameter(':nextWeek', DateTime::from('+ 7 days'))
             ->groupBy('t.id')
+            ->addGroupBy('tt.startDateTime')
```

Comment:
Is the `->groupBy('t.id')` above needed?

Reply:
TBH i dont know, i did not try it without that, was just fixing the error in mysql

Reply:
Could you try?